src/NLog.MailKit/MailTarget.cs

```diff
@@ -522,16 +522,40 @@ private MimeMessage CreateMailMessage(LogEventInfo lastEvent, string body)
 
             if (Priority != null)
             {
-                var renderedPriority = Priority.Render(lastEvent);
-                try
-                {
+                // default is already set by MimeMessage ctor
+                //msg.Priority = MessagePriority.Normal;
```

Comment:
This looks like dead code, maybe the comment should be

// default (MessagePriority.Normal) is already set by MimeMessage ctor